### ISSUE TITLE
Regression in version 2.2.1 and 2.2.2

### ISSUE THREAD

**Issue by dfaust** [Canop/deser-hjson#20]
Hi. The following document can be parsed with version 2.2.0 and below, but not with version 2.2.1 or 2.2.2.

```rust
        indoc! {r#"
            {
                dirs: [
                ]
            }
        "#},
```

Produces the string:

`{\n    dirs: [\n    ]\n}`

Which causes the error:

`Error: ExpectedMap at 3:5 at "]\n}`

I have a bunch of other cases in my test suite, so there seems to be a general issue (maybe because of the line breaks?).

**Comment by Canop** [comment]
As I don't have your full test case, I can't be sure. Can you please check it's solved with PR https://github.com/Canop/deser-hjson/pull/21 ?

**Comment by dfaust** [comment]
Yes, it works.
Thanks for this very quick fix!

**Comment by Canop** [comment]
Fix released in version 2.2.3
